openedx/core/djangoapps/certificates/api.py

```diff
@@ -10,28 +10,11 @@
 
 
 def auto_certificate_generation_enabled():
-    return (
-        SWITCHES.is_enabled(waffle.SELF_PACED_ONLY) or
-        SWITCHES.is_enabled(waffle.INSTRUCTOR_PACED_ONLY)
-    )
-
-
-def auto_certificate_generation_enabled_for_course(course):
-    if not auto_certificate_generation_enabled():
-        return False
-
-    if course.self_paced:
-        if not SWITCHES.is_enabled(waffle.SELF_PACED_ONLY):
-            return False
-    else:
-        if not SWITCHES.is_enabled(waffle.INSTRUCTOR_PACED_ONLY):
-            return False
-
-    return True
+    return SWITCHES.is_enabled(waffle.AUTO_CERTIFICATE_GENERATION)
 
 
 def _enabled_and_self_paced(course):
```

Comment:
Hah, I got this function name completely wrong!  This should really be called `_enabled_and_instructor_paced(course)`.

🤦‍♂️